.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules
 **/.DS_Store
 .idea
 telemetryCache
+welcomePage.html
```

Comment:
Curiously, why don't you commit this file?

Reply:
At this point, this file shouldn't be hand-edited. It should only be generated from the `extension-readme.md` file.

Thinking that we'll want to keep this in the future even if we move away from `extension-readme.md` as the base since Markdown is easier to modify...the only reason why we're transpiling to HTML is because VS Code can't render Markdown without manually opening a preview window.

Reply:
Gotcha, thanks for the explain.